### PR TITLE
Fix create_node_allocation

### PR DESCRIPTION
**Pull request by brodokk**
If you look at the  [Pterodactyl documentation](https://dashflo.net/docs/api/pterodactyl/v1/) `Create allocations` the API will return a 204 with no content

### CODE REVIEW

File: pytero/app.py [hunk 1]
```diff
@@ -632,7 +632,18 @@ async def create_node_allocation(
...
+        """ Create a new node.
+
+        node: :class:`int`
+            The ID of the node.
+        ip: :class:`str`
+            The ip of the node.
+        ports: :class:`list[str]`
+            A list of ports to be used by the node.
+        alias: :class:`str`
+            Alias name of the node.
```
Comment (devnote-dev) [inline pytero/app.py]:
```suggestion
            The IP for the allocation.
        ports: :class:`list[str]`
            A list of ports or port ranges for the allocation.
        alias: :class:`str`
            Alias name of the node.
```

File: pytero/app.py [hunk 2]
```diff
@@ -632,7 +632,18 @@ async def create_node_allocation(
...
+        """
         data = await self._http.post(
```
Comment (devnote-dev) [inline pytero/app.py]:
```suggestion
        await self._http.post(
```

**Review by devnote-dev — CHANGES_REQUESTED** [review]

**Comment by brodokk** [comment]
Hey, thanks for your propositions. I actually update a bit more what you was proposing for the documentation part to make it more consistent.

File: pytero/app.py [hunk 3]
```diff
@@ -632,17 +632,26 @@ async def create_node_allocation(
         ip: str,
         ports: list[str],
         alias: str = None
-    ) -> Allocation:
-        data = await self._http.post(
+    ) -> None:
+        """ Create a new allocated node.
```
Comment (devnote-dev) [inline pytero/app.py]: It's creating a node allocation, not a node.
```suggestion
        """ Create a new node allocation.
```

File: pytero/app.py [hunk 4]
```diff
@@ -632,17 +632,26 @@ async def create_node_allocation(
...
+
+        node: :class:`int`
+            The ID of the allocation.
```
Comment (devnote-dev) [inline pytero/app.py]: This isn't the allocation ID, it's the node ID. The allocation ID wouldn't exist yet.

**Review by devnote-dev — CHANGES_REQUESTED** [review]

**Comment by brodokk** [comment]
Sorry for this mistakes, should be good now

**Review by devnote-dev — APPROVED** [review]

**Pull request merged**